tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -46,7 +46,14 @@
         <!-- The directory in which the additional ICU XML data is stored. -->
         <property name="specialsDir" value="${basedir}/../../../icu4c/source/data/xml"/>
 
-        <!-- An override for the CLDR version string to be used in generated data. -->
+        <!-- Default value for ICU version (icuver.txt). Update this for each release. -->
```

Comment:
Oh, one quick thought -- do we want to also add this to the steps on the page here as well:
http://site.icu-project.org/processes/release/tasks/versions

Reply:
Yes. The section "icu4c/source/data/misc/icuver.txt needs to be updated ..." needs to be updated to say that this xml file here needs to be updated, and that the .txt file is now generated.

Reply:
I updated the release task description. Please fix if I got it wrong.

Reply:
Thanks Markus!
FWIW, it looks good to me -- thanks for updating it!
Also, cc: @erik0686 and @daniel-ju, since I think they'll be working on the version number updates.